src/migration/1574330659061-UpdateUserEntityDateOfBirth.ts

```diff
@@ -0,0 +1,11 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class UpdateUserEntityDateOfBirth1574330659061 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<any> {
+    }
+
+}
```

Comment:
For alle 3 migrations: Er de nødvendige at have med? De ændrer jo intet

Reply:
Næh det er de vel ikke. Troede de ændrede noget men de gør intet

Reply:
Har fjernet dem nu